tests/posix/pthread_pressure/Kconfig

```diff
@@ -15,7 +15,6 @@ config TEST_NUM_CPUS
 
 config TEST_DURATION_S
 	int "Number of seconds to run the test"
-	range 1 21600
 	default 5
 	help
 	   Duration for the test, in seconds. The range has a reblatively high
```

Comment:
We drop the range, but still mention its values in help.

Reply:
Fixed!